x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/new_search_index_template.tsx

```diff
@@ -95,58 +100,81 @@ export const NewSearchIndexTemplate: React.FC<ISearchIndex> = ({
           </EuiText>
         </EuiFlexItem>
         <EuiFlexItem grow>
-          <EuiFormRow
-            label={i18n.translate(
-              'xpack.enterpriseSearch.content.newIndex.newSearchIndexTemplate.nameInputLabel',
-              {
-                defaultMessage: 'Name your {type}',
-                values: { type: type.toLowerCase() },
-              }
-            )}
-            fullWidth
-          >
-            <EuiFieldText
-              placeholder={i18n.translate(
-                'xpack.enterpriseSearch.content.newIndex.newSearchIndexTemplate.nameInputPlaceholder',
+          <EuiFlexGroup>
+            <EuiFlexItem grow>
+              <EuiFormRow
+                label={i18n.translate(
+                  'xpack.enterpriseSearch.content.newIndex.newSearchIndexTemplate.nameInputLabel',
+                  {
+                    defaultMessage: 'Name your {type}',
+                    values: { type },
+                  }
+                )}
+                fullWidth
+              >
+                <EuiFieldText
+                  placeholder={i18n.translate(
+                    'xpack.enterpriseSearch.content.newIndex.newSearchIndexTemplate.nameInputPlaceholder',
+                    {
+                      defaultMessage: 'Set a name for the {type}',
+                      values: { type },
+                    }
+                  )}
+                  fullWidth
+                  isInvalid={false}
+                  value={rawName}
+                  onChange={(event) => handleNameChange(event)}
```

Comment:
You should just be able to do `onChange={handleNameChange}` as shorthand for this (same for https://github.com/elastic/kibana/pull/131733/files#diff-8eedef514e931c0b02a25d3c28ec370cd2969d1f7e289bb8a4172b5b67d512aaR141)